br/pkg/restore/stream_metas_test.go

```diff
@@ -246,6 +247,46 @@ func TestTruncateSafepoint(t *testing.T) {
 	}
 }
 
+func TestTruncateSafepointForGCS(t *testing.T) {
```

Comment:
do we need to test azblob storage ?

Reply:
Currently, the unit test and integration test for azure are not supported.